resources/styles/components/task-step/reading/book-content/work-in-physics.less

```diff
@@ -28,4 +28,16 @@
 
   }
 
+  figure {
+    width: 100%;
+    margin: 0;
+    img {
+      width: 100%;
+    }
+  }
+
+  div[data-type="title"] + figure + p {
```

Comment:
`> p:first-child` doesn't work because it's not a direct descendant of the parent `.work-in-physics`.  `first-of-type` does though, switching to that